pkg/compiler/debug.go

```diff
@@ -26,7 +26,9 @@ type DebugInfo struct {
 	Hash      util.Uint160      `json:"hash"`
 	Documents []string          `json:"documents"`
 	Methods   []MethodDebugInfo `json:"methods"`
-	Events    []EventDebugInfo  `json:"events"`
+	// NamedTypes are structured types that have some name and a number of internal fields.
```

Comment:
They're all named in the resulting config file and they can't be anonymous, but in the compiler there is some difference, I'll try to check how it can be handled.